Update ExcelDocument.cs

lukasbelak opened the pull request:
fixed nullReference exception when user wants to generate metadata for foreign language and his translations are incompleted (missing translations)

MscrmTools commented:
Can you just explain what you changed exactly?
Formatting has changed in the file so it is hard to identify changes

lukasbelak commented:
I just added checks like 

if (object != null) 
then 
var something = object.property;

my client has a german language installed but incomplete translations and there were nullReference exceptions, so I added a few checks and changed First() (which was used in all over the file) method to FirstOrDefault()
